docs/resources/vmalertmanagerconfig.md

```diff
@@ -0,0 +1,99 @@
+# VMAlertmanagerConfig
+
+The `VMAlertmanagerConfig` provides way to configure [VMAlertmanager](./vmalertmanager.md) 
+configuration with CRD. It allows to define different configuration parts, which will be merged by operator into config. 
+
+It behaves like other config parts - `VMServiceScrape` and etc.
+
+## Specification
+
+You can see the full actual specification of the `VMAlertmanagerConfig` resource in 
+the **[API docs -> VMAlertmanagerConfig](../api.md#vmalertmanagerconfig)**.
+
+Also, you can check out the [examples](#examples) section.
```

Comment:
Wdyt about adding a caution about special case here?
Something like "read this before using". I expect this will increase chances that users will find out about this limitation before actually facing this and researching docs again to see what went wrong.

Reply:
done